libs/wingsdk/src/target-tf-aws/api.ongetrequest.inflight.ts

```diff
@@ -0,0 +1,14 @@
+import { IApiOnGetHandlerHandlerClient } from "./api";
+import type { IFunctionHandlerClient } from "../cloud";
+
+export class ApiOnGetRequestHandlerClient
+  implements IApiOnGetHandlerHandlerClient
+{
+  private readonly handler: IFunctionHandlerClient;
+  constructor({ handler }: { handler: IFunctionHandlerClient }) {
+    this.handler = handler;
+  }
+  public async handle(event: any) {
```

Comment:
Can we add type information for the `any` here? I think the types are available in `@types/aws-lambda`, check out the code here: https://docs.aws.amazon.com/lambda/latest/dg/typescript-handler.html#async-typescript

This class will need to transform the AWS request input into an cloud-agnostic `cloud.ApiRequest` struct, and then transform the `cloud.ApiResponse` returned by `handler.handle(event)` back into AWS response event object.